src/MCMCChains.jl

```diff
@@ -2,9 +2,10 @@ module MCMCChains
 
 using AxisArrays
 const axes = Base.axes
-
+using AbstractFFTs
 import AbstractMCMC
-import AbstractMCMC: chainscat
+import AbstractMCMC: chainscat, AbstractChains
```

Comment:
I don't think we use `AbstractChains` anywhere? Probably it's just left from the initial PR:
```suggestion
import AbstractMCMC: chainscat
```